models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -1,6 +1,7 @@
 {{  config(
         alias='trades',
         materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
In the `cow_protocol_ethereum_trades.sql` model, the block_date column is not in the `trades_with_token_units`CTE or in the `valued_trades` one (maybe missing in other ones as well)

Reply:
But those are just preliminary tables... am I supposed to put it everywhere?

Reply:
OH! I see I only put it at the top and didn't trickle down to the actual result. Whoops.

Reply:
Yes exactly :)

Reply:
Fixed!